sql/src/main/java/io/crate/lucene/LuceneQueryBuilder.java

```diff
@@ -1088,10 +1088,12 @@ private static Query getPolygonQuery(Geometry geometry, IndexGeoPointFieldData f
 
             private Query getBoundingBoxQuery(Shape shape, IndexGeoPointFieldData fieldData) {
                 Rectangle boundingBox = shape.getBoundingBox();
-                return new LegacyInMemoryGeoBoundingBoxQuery(
-                    new GeoPoint(boundingBox.getMaxY(), boundingBox.getMinX()),
-                    new GeoPoint(boundingBox.getMinY(), boundingBox.getMaxX()),
-                    fieldData
+                return new GeoPolygonQuery(
```

Comment:
I didn't know the index format might have changed. Will revert and investigate separately.